### DISCUSSION TITLE
Block promotional mails

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1918] [Feature Requests]
Hi!

Please add something like [https://support.mozilla.org/en-US/kb/enable-block-promotional-emails-firefox-relay-prem](this) feature in Firefox Relay. Preferably to have a toggle to automatically put such mails in SPAM or reject at all.

**Comment by ghost** [comment]
Good idea!
@nguyenkims can it be realised?

**Reply by nguyenkims** [reply]
Is it similar to https://github.com/simple-login/app/discussions/542 ?

**Reply by ghost** [reply]
@nguyenkims i don’t think so :)
I think person means that promoting mails (AKA marketing campaigns) can be blocked by SL
